TicTacToe2/Controller/EventController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using TicTacToe2.View;
+
+namespace Script
+{
+    public class EventController
+    {
+        private static EventController thisInstance = null;
+        private static readonly object padlock = new object();
+
+        private SortedDictionary<string, Action<string[]>> eventList = new SortedDictionary<string, Action<string[]>>();
```

Comment:
Create a custom data structure
Need :
-  Multiple identic key, so a get could return multiple function
- Being fast for looking